docs/mesa.md

```diff
@@ -1,11 +1,5 @@
 # mesa package
 
-## Subpackages
-
-```{toctree}
-mesa.visualization_old
```

Comment:
Should we add the new visualisation here? It's nice to have that build in the docs, even if we don't have that much docstring.

Reply:
That can happen in a separate PR. This PR is rather expensive to maintain/rebase.